Doc/c-api/object.rst

```diff
@@ -492,6 +492,13 @@ Object Protocol
    on failure.  This is equivalent to the Python statement ``del o[key]``.
 
 
+.. c:function:: int PyObject_DelItemString(PyObject *o, const char *key)
+
+   This is the same as :c:func:`PyObject_DelItem`, but *key* is
+   specified as a :c:expr:`const char*` UTF-8 encoded bytes string,
```

Comment:
```suggestion
   This is the same as :c:func:`PyObject_DelItem`, but *key* must be
   a null-terminated :c:expr:`const char*` UTF-8 encoded bytes string,
```

Reply:
I think this is clearer and more precise.

Reply:
Not disagreeing, but this is just copy-paste from the surrounding `PySomething_DoSomethingString` functions. I'd rather we be consistent.

Reply:
Let's be consistent and keep what you already wrote. If we want to mention NULL terminated `const char *`, then we should instead change everything in the docs in one go, but not in this PR (and I don't think we should do it in the first place)